dist/FxAccountsPairingChannel.babel.umd.js

```diff
@@ -1208,13 +1208,13 @@ function bytesAreEqual(v1, v2) {
     return false;
   }
 
+  var mismatch = false;
+
   for (var i = 0; i < v1.length; i++) {
-    if (v1[i] !== v2[i]) {
-      return false;
-    }
+    mismatch &= v1[i] !== v2[i];
   }
 
-  return true;
+  return !mismatch;
```

Comment:
I don't know why this is in the build files, maybe `npm run build` didn't get run correctly in a previous change to master.